app/controllers/administrator/products_controller.rb

```diff
@@ -0,0 +1,49 @@
+class Administrator::ProductsController < Administrator::BaseController
```

Comment:
I think the term `admin` is more commonly used instead of `administrator`, but I guess it's fine for now :-)

Reply:
solved. ✓😄
am folosit asa, sa nu fie considerat prescurtare :-?
rezolvat oricum :D